docs/model-dev-guide/dtrain/dtrain-implement.rst

```diff
@@ -219,40 +228,38 @@ batch inference job, create a new PyTorchTrial and follow these steps:
 Once the new PyTorchTrial object is created, use the experiment configuration to distribute
 inference in the same way as training. `cifar10_pytorch_inference
 <https://github.com/determined-ai/determined/blob/master/examples/computer_vision/cifar10_pytorch_inference/>`_
-is an example of distributed batch inference.
+serves as an example of distributed batch inference.
 
 .. _config-template:
 
 *************************
  Configuration Templates
```

Comment:
The entire **Configuration Templates** sections seems misplaced on this page. It is not distributed-training-specific advice (though it is helpful information, which I think deserves its own page).

Reply:
done